web-common/src/features/dashboards/filters/measure-filters/measure-filter-options.ts

```diff
@@ -73,3 +77,17 @@ export const MeasureFilterOptions: MeasureFilterOption[] = [
     shortLabel: "",
   },
 ];
+// Full list with options not supported in filter pills just yet.
```

Comment:
Mind creating a GitHub issue to track this work?

Reply:
I am not sure if this is the end goal. But @mindspank is aware of this, dont think it makes sense to create an issue until we know what we are doing.

Reply:
In the Criteria section of the Alert Metadata, I see that the Criterion pill doesn't handle e.g. the "Does not equal" operator. In the pic below, rather than seeing "New vaccinations != [some number]", it just says "New vaccinations". 

<img width="1079" alt="image" src="https://github.com/rilldata/rill/assets/14206386/3cd84f17-b298-4fbb-a996-6aa4c1c494f7">

I assumed the code comment referred to supporting all operators in the Criterion pills, so the Alert Metadata can showcase all kinds of Criteria.